packages/framework-core/src/services/read-model-store.ts

```diff
@@ -144,27 +324,29 @@ export class ReadModelStore {
       },
     } as BoosterMetadata
     logger.debug(
-      `[ReadModelStore#project] Storing new version of read model ${readModelName} with ID ${readModelID}:`,
+      `Storing new version of read model ${readModelName} with ID ${readModelID}, version ${newReadModel.boosterMetadata.version} and expected database version ${expectedCurrentDatabaseVersion}:`,
       newReadModel
     )
-
-    return this.config.provider.readModels.store(this.config, readModelName, newReadModel, currentReadModelVersion)
+    return this.config.provider.readModels.store(
+      this.config,
+      readModelName,
+      newReadModel,
+      expectedCurrentDatabaseVersion
+    )
   }
 
   /**
    * Gets a specific read model instance referencing it by ID when it's a regular read model
    * or by ID + sequenceKey when it's a sequenced read model
    */
-  public async fetchReadModel(
+  async fetchReadModel(
```

Comment:
is this no longer public because it was replaced by `getReadModels` to implement the new features? Don't we need to call this from outside this file?

Reply:
It is only called outside in unit tests; that's why it was left without the `public` but not made `private`.